app/views/well_known/webfinger/show.json.rabl

```diff
@@ -8,9 +8,9 @@ end
 
 node(:links) do
   [
-    { rel: 'http://webfinger.net/rel/profile-page', type: 'text/html', href: account_url(@account) },
+    { rel: 'http://webfinger.net/rel/profile-page', type: 'text/html', href: short_account_url(@account) },
     { rel: 'http://schemas.google.com/g/2010#updates-from', type: 'application/atom+xml', href: account_url(@account, format: 'atom') },
-    { rel: 'self', type: 'application/activity+json', href: account_url(@account) },
+    { rel: 'self', type: 'application/activity+json', href: account_url(@account, format: 'json') },
```

Comment:
The ActivityPub self link needs no format because the format is supposed to depend on request header.